### PR TITLE
add inital changes

### PR DESCRIPTION
**Pull request by mrlukyman**

### CODE REVIEW

File: src/components/AppPreview.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+import React from "react"
+import "../assets/White.png"
```
Comment (effektsvk) [inline src/components/AppPreview.js]: You are not using this import. Store it into a variable, e.g.: `import white from "../assets/White.png"` and then use it in `src={white}`

Comment (effektsvk) [reply src/components/AppPreview.js]: [This](https://github.com/mrlukyman/city-cube/blob/35df801bb9cc1db2ea440416735fd61f97d2b448/src/components/Background.js#L2) is correct implementation.

File: src/components/Background.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import React from "react"
+import background from "../assets/background.png"
+
+export const Background = () => (
+    <img src={background} alt="background"
+    style={{
+        width:"100%", 
+        height:"100",
```
Comment (effektsvk) [inline src/components/Background.js]: Is this supposed to be `100%` or `100px`?

File: src/index.css [hunk 3]
```diff
@@ -14,30 +14,45 @@ code {
...
-  margin: 25px 0 0 100px;
+.navbar {
   display: flex;
   align-items: center;
+  justify-items: center;
   flex-direction: row;
-}
-
-
+  margin-left: auto; /* fix this shit to flex */
```
Comment (effektsvk) [inline src/index.css]: Add `TODO:` prefix into comment so you can easily find this comment with [Todo Tree](https://marketplace.visualstudio.com/items?itemName=Gruntfuggly.todo-tree) VS Code extension.

File: src/components/Background.js (hunk 2 above)
Comment (mrlukyman) [reply src/components/Background.js]: %

File: src/components/AppPreview.js [hunk 4]
```diff
@@ -1,6 +1,27 @@
...
+    left: "1",
+    right: "2"
+}
+
+const Image = styled.img`
+    display: flex;
+    width: ${(props) => size[props.size]};
+    justify-content: flex-end;
+    padding-top: 40px;
+    order: ${(props) => order[props.order]};
```
Comment (effektsvk) [inline src/components/AppPreview.js:20]: use `flex-direction` instead of `order`

**Review by effektsvk — APPROVED** [review]

**Pull request merged**